### DISCUSSION TITLE
Any way to build by linking to an external tree-sitter (without using the submodule)?

### DISCUSSION THREAD

**Discussion by bbigras** [afnanenayet/diffsitter#163] [Q&A] [answered]
I want to package diffsitter for nixos, and I was asked to use the already packaged tree-sitter instead of the submodule.

I'm guessing I would need to patch `build.rs`.

Any ideas?

**Comment by afnanenayet** [comment, marked as answer]
This is currently in development. You can track through #167 and #177

**Comment by afnanenayet** [comment]
This has been implemented
